### DISCUSSION TITLE
PyDPF Post with LSTC License

### DISCUSSION THREAD

**Discussion by callaway-nathan-robbins** [ansys/pydpf-post#631] [Q&A] [answered]
Does PyDPF Post work with an existing LSTC license?

**Comment by PProfizi** [comment, marked as answer]
Hello @callaway-nathan-robbins,

The licensing requirements are explicited in the PyDPF-Core documentation [here](https://dpf.docs.pyansys.com/version/dev/getting_started/licensing.html#compatible-ansys-license-increments).
It states that acceptable license increments linked to LS-Dyna are:
- `dynapp` available in the `ANSYS LS-DYNA PrepPost` product
- `dyna` available in the `ANSYS LS-DYNA` product

Does that answer the question?

**Comment by callaway-nathan-robbins** [comment]
Got it, thanks.
